pybossa/cache/projects.py

```diff
@@ -105,7 +105,7 @@ def format_task(row, lock_users=[]):
     # avoid sql planer doing indexscan when searching through table,
     # the settings is LOCAL and only applies to current transaction,
```

Comment:
The comments here are obsolete now and can be removed.

Reply:
This comment still applies to the Task Browse sql, we still need to disable index scan for Task Browse query to get a better performance.

Reply:
got it, thanks!